t/t0002-request.t

```diff
@@ -106,14 +106,14 @@ test_expect_success 'request: rpc test client works with no request payload' '
 		test -s attr.list.out
 '
 test_expect_success 'request: rpc test client works with request payload' '
-	$jq -j -c -n  "{name:\"rank\"}" | $RPC attr.get >attr.get.out &&
```

Comment:
Tests are using the `$jq` variable because it is set in `flux-sharness.sh`:

https://github.com/flux-framework/flux-core/blob/56c98db68acee51db413ecf9d53f2e4588bf378b/t/sharness.d/flux-sharness.sh#L353

It does seem like a good idea to consistently use `jq` directly instead of `$jq` because I'm sure lots of places already weren't, but the line above where `$jq` is set should also be dropped.